cellmlmanip/model.py

```diff
@@ -410,27 +410,44 @@ def get_equation_graph(self, refresh=False) -> nx.DiGraph:
         self.graph = graph
         return graph
 
-    def get_equations_for(self, symbols, lexicographical_sort=True):
+    def get_equations_for(self, symbols, sort_by_input_symbols=False, excluded_symbols=[]):
```

Comment:
Normally I would say it was not ideal to change the function signature - but since it is entirely internal I think its okay.

Reply:
Yes good idea feel free to change :)